content/docs/_index.md

```diff
@@ -1,55 +1,23 @@
 ---
 title: Documentation
+url: "docs"
 ---
 
-The Odin programming language is fast, concise, readable, pragmatic and open sourced. It is designed with the intent of replacing C with the following goals:
+## Getting Started
 
-* simplicity
-* high performance
-* built for modern systems
-* joy of programming
+1. [Install the Odin compiler and libraries](/docs/install).
 
-## Installing Odin
+2. Take a [tour of the Odin language](/docs/overview).
 
-### [Getting Started](/docs/install)
+## Resources
 
-Instructions for downloading and install the Odin compiler and libraries.
+* [Frequently Asked Questions (FAQ)](/docs/faq) has answers to common questions about Odin.
 
+* [The Odin Wiki](https://github.com/odin-lang/Odin/wiki) is a community-maintained resource.
 
-## Learning Odin
-
-### [Overview of Odin](/docs/overview)
-
-An overview of the Odin programming language.
-
-### [Frequently Asked Questions (FAQ)](/docs/faq)
-
-Answers to common questions about Odin.
-
-### [The Odin Wiki](https://github.com/odin-lang/Odin/wiki)
-
-A wiki maintained by the Odin community.
-
-### [Odin Discord](https://discord.gg/sVBPHEv)
-
-Get live support and talk with other odiners on the Odin Discord.
-
-### [Demonstration Code Example](/docs/code-example)
-
-A demonstration of the language in Odin code.
-
-## References
-
-### [Language Specification](/ref/spec)
-
-The official Odin Language specification.
-
-## Articles
-
-### [The Odin Blog](/blog)
-
-The official blog of the Odin programming language, featuring announcements, news, and in-depth articles by the Odin team and guests.
```

Comment:
Why is this removed?

Reply:
That's not removed. I'm not sure why it says that it is.